scripts/render-change

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+"""Generate a changelog file.
+
+Usage
+=====
+
+::
+
+    $ scripts/gen-change --format rst
+    # Or:
+    $ scripts/gen-change --format single-html -r 2.0.0
+
+
+"""
+import os
+import sys
+import json
+import argparse
+
+
+ROOTDIR = os.path.dirname(
+    os.path.dirname(os.path.abspath(__file__)))
+CHANGEDIR = os.path.join(ROOTDIR, '.changes')
+SINGLE_HTML_TEMPLATE = """
+%s
+
+.. list-table::
+   :header-rows: 1
+
+%s
+
+"""
+
+
+def sorted_releases(changedir):
+    def key(filename):
+        filename = filename[:-len('.json')]
+        parts = tuple(int(i) for i in filename.split('.'))
+        return parts
+    releases = sorted(os.listdir(changedir), key=key, reverse=True)
+    return releases
+
+
+def _generate_single_release_rst(release_identifier, data, contents):
+    contents.append(release_identifier)
+    contents.append('=' * len(release_identifier))
+    contents.append('')
+    for change in data:
+        line = '* %s:%s: %s' % (change['type'], change['category'],
+                                change['description'])
+        contents.append(line)
+    contents.append('')
+    contents.append('')
+
+
+def generate_rst_changelog(args, changedir=CHANGEDIR):
+    if not os.path.isdir(changedir):
+        sys.stderr.write("Changedir does not exist: %s\n" % changedir)
+        return
+    contents = []
```

Comment:
We should include the ``CHANGELOG`` header at the top if you are planning to completely trying to reprint the changelog from scratch.